recipes/unidist/meta.yaml

```diff
@@ -0,0 +1,130 @@
+# https://github.com/modin-project/unidist/blob/0.1.0/setup.py#L39-L55)
+# from "unidist[<option>]" to "unidist-<option>" as a conda-forge package
+outputs:
+  - name: unidist-all
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('unidist', exact=True) }}
+        - {{ pin_subpackage('unidist-dask', exact=True) }}
+        - {{ pin_subpackage('unidist-ray', exact=True) }}
+        - {{ pin_subpackage('unidist-mpi', exact=True) }}
+    test:
+      imports:
+        # dummy test; actual tests are in subpackages
+        - unidist
+
+  - name: unidist
+    build:
+      noarch: python
+      script: python -m pip install . --ignore-installed -vv
+    requirements:
+      host:
+        - pip
+        - python
+      run:
+        - python
+        - packaging
+        - cloudpickle
+    test:
+      imports:
+        - unidist
+      commands:
+        # unidist has multiprocessing and python backends by default so we test both
+        - python -c "import unidist; import unidist.config as cfg; cfg.Backend.put('python'); unidist.init(); ref = unidist.put(1); unidist.get(ref)"
+        - python -c "import unidist; import unidist.config as cfg; cfg.Backend.put('multiprocessing'); unidist.init(); ref = unidist.put(1); unidist.get(ref)"
+
+  - name: unidist-dask
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('unidist', exact=True) }}
+        - dask >=2.22.0
+        - distributed >=2.22.0
+    test:
+      imports:
+        - unidist
+      commands:
+        - python -c "import unidist; import unidist.config as cfg; cfg.Backend.put('dask'); unidist.init(); ref = unidist.put(1); unidist.get(ref)"
+
+  - name: unidist-ray
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('unidist', exact=True) }}
+        - ray-default >=1.4.0
+        # aioredis 2.x breaks Ray
+        # ref: https://github.com/conda-forge/ray-packages-feedstock/issues/38
+        - aioredis <2
+    test:
+      imports:
+        - unidist
+      commands:
+        - python -c "import unidist; import unidist.config as cfg; cfg.Backend.put('ray'); unidist.init(); ref = unidist.put(1); unidist.get(ref)"
+
+  - name: unidist-mpi
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('unidist', exact=True) }}
+        - mpi4py >=3.0.3
+        - msgpack-python
+    test:
+      # We do not run actual tests for mpi backend because
+      # those are currently failing on Windows.
+      # See more in https://github.com/modin-project/unidist/issues/69.
+      # We should run the tests after the issue is fixed.
+      imports:
+        - unidist
+
+about:
+  home: https://github.com/modin-project/unidist
+  license: Apache-2.0
```

Comment:
As you are also including a part of versioneer, its license and license file also need to be included.

Ref: https://github.com/modin-project/unidist/blob/master/versioneer.py

Reply:
The license of versioneer is Public Domain, which has no copyright https://creativecommons.org/publicdomain/zero/1.0/.

I haven't seen any conda recipe that would mentioned about it so I think we can omit this. Does this make sense?